configuration_go/schemas/thanos/reqlogging/reqlogging.go

```diff
@@ -17,6 +19,15 @@ type RequestConfig struct {
 	Options OptionsConfig       `yaml:"options,omitempty"`
 }
 
+func (c RequestConfig) String() string {
+	// We use "gopkg.in/yaml.v2" instead of "github.com/ghodss/yaml" for correct formatting of this config.
```

Comment:
This comment can be put on the function as a goDoc comment, so that user can see it?

Reply:
done